CODE_OF_CONDUCT.md

```diff
@@ -39,5 +39,6 @@ Instances of abusive, harassing, or otherwise unacceptable behavior
 may be reported by opening an issue
 or contacting one or more of the project maintainers.
 
-This Code of Conduct is adapted from the [Contributor Covenant](http://contributor-covenant.org), version 1.2.0,
-available at [http://contributor-covenant.org/version/1/2/0/](http://contributor-covenant.org/version/1/2/0/)
+This Code of Conduct is adapted from the [Contributor
```

Comment:
Do we intend to keep the same code of conduct? Not blocking this PR though.

Reply:
I think we'd remove it as part of the google, istio etc. clean up. Then use our [contributing markdown file](https://github.com/open-telemetry/opentelemetry-python/blob/main/CONTRIBUTING.md#contributing-to-opentelemetry-python) as the "code of conduct" equivalent. I looked through other repos & didn't notice a code of conduct specifically